lib/control-plane/kustomization.yaml

```diff
@@ -10,6 +10,19 @@ secretGenerator:
     options:
       disableNameSuffixHash: true
 
+  - name: libvirt-secret
+    behavior: create
+    literals:
+      - LibvirtPassword=12345678
+    options:
+      disableNameSuffixHash: true
+
+  - name: octavia-ca-passphrase
+    literals:
+      - server-ca-passphrase=12345678
+    options:
+      disableNameSuffixHash: true
```

Comment:
Actually, one last thing.  Please see this comment [1] on another DT PT and let me know what you think.

[1] https://github.com/openstack-k8s-operators/architecture/pull/159/files#r1570900110

Reply:
Makes sense, let me move this to `dt` specific change.